test/e2e/agent_dir/newrelic-infra-agent/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM newrelic/infrastructure-bundle:latest
```

Comment:
why the bundle? i think `infrastructure` should be enough since the rest of the integrations are not needed

Reply:
You're right, infra image is enough